### DISCUSSION TITLE
failed install from beta 79 to beta 80

### DISCUSSION THREAD

**Discussion by cfritze64** [johnlindquist/kit#98] [Errors] [answered]
Ran the beta80 installer and get a window that flashes on the screen -very- briefly. I think the title is "ScriptKit failed to install", but it's at the limit of my perception, it's up and gone so quickly. Anywhere I can look for a log?

**Comment by johnlindquist** [comment, marked as answer]
Q: Is the auto-update not working for you? It should "silently" try to update each time the computer wakes from sleep.

The install logs are at (I'm out on my phone, only 90% sure on this):

~/Library/Logs/Kit/

**Reply by cfritze64** [reply]
Aha! didn't know there were autoupdates going on. Or maybe they are broken on my machine. In any case, I think I'll have to copy out my scripts and consider a clean re-installation as the beta80 won't run fully and Kit isn't installed properly either.

**Reply by cfritze64** [reply]
(there is a Logs/Kit folder, but the log file, while timestamped with today's attempts, is a zero bytes empty file.)

**Reply by johnlindquist** [reply]
Before copying out your scripts, try just deleting `~/.kit`. I should only [try to setup ~/.kenv](https://github.com/johnlindquist/kitapp/blob/beta/src/main.dev.ts#L391) if the directory doesn't exist.

**Reply by cfritze64** [reply]
Terrific. thank you!. deleting ~/.kit and then running the beta80 installer worked great. I'll try to pay more attention to the update text in the interface when you build new betas to confirm they're coming across.

**Reply by johnlindquist** [reply]
The text is just [Updating/Relaunching](https://github.com/johnlindquist/kitapp/blob/beta/src/main.dev.ts#L70)

It auto-quits, installs, and relaunches. I probably need to add a way to disable auto-update in the options...
